javascript/ql/test/library-tests/Portals/src/typeTracking/simpleField.js

```diff
@@ -0,0 +1,12 @@
+// new portal exit node (i.e. caught with typetracking) annotated below
+
+var i = require('./index').NP;
+var np = new i();
+
+np.argFunc( input => {
+	
+	input.on('arg', x => {
+	input.emit( { x: "hello", y: input.field}); // this is a portal exit node, since input is an
```

Comment:
Which expression does "this" refer to? And can you briefly explain why the portal exit node can only be identified with type tracking?

Reply:
Thanks, good catch! I meant the "input.field" expression, but it was actually caught with the original portal exit node tracking. My bad.

New commit removes this test.